game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/BattleSteps.java

```diff
@@ -44,9 +55,14 @@
   final @NonNull Supplier<Collection<Territory>> getAttackerRetreatTerritories;
   final @NonNull BiFunction<GamePlayer, Collection<Unit>, Collection<Territory>>
       getEmptyOrFriendlySeaNeighbors;
+  final @NonNull BattleActions battleActions;
 
   public List<String> get() {
 
+    final BattleStep submergeSubsVsOnlyAir = new SubmergeSubsVsOnlyAirStep(this, battleActions);
+    final BattleStep airAttackVsNonSubs = new AirAttackVsNonSubsStep(this, battleActions);
+    final BattleStep airDefendVsNonSubs = new AirDefendVsNonSubsStep(this, battleActions);
```

Comment:
As I mentioned in the first comment, these manual instantiations of the steps will be moved into a step factory that builds the required steps and puts them in the correct order.  But I can't build that factory until I have a few more steps converted.